gym/envs/__init__.py

```diff
@@ -758,7 +757,7 @@ def _merge(a, b):
     "yars_revenge",
     "zaxxon",
 ]:
-    for obs_type in ["image", "ram"]:
+    for obs_type in ["rgb", "ram"]:
```

Comment:
`ALGymEnv` still accepts `image` as an obs_type but will display a deprecation warning asking the user to switch to one of: `rgb` or `grayscale`.